client/src/ui/molecules/language-menu/index.scss

```diff
@@ -0,0 +1,34 @@
+@import "~@mdn/minimalist/sass/vars/layout";
+
+.language-menu {
```

Comment:
![image](https://user-images.githubusercontent.com/3604775/94942516-b3bdb980-04d6-11eb-9732-bf7109245f7a.png)

Just checking that it's intended that the items are not yet aligned.

Reply:
On which OS browser combo did you see this? That is one of the tricky things with things like the `select` element. There is very little control we have over it via CSS, but what we get in terms of accessibility is hard to beat.

Reply:
I tested in Fx and Chrome on Mac as can be seen in the screenshots.